test/api/publish.test.js

```diff
@@ -152,3 +157,29 @@ describe('publishing endpoints', () => {
     );
   });
 });
+
+describe('API versioning', () => {
+  function getPreconfiguredRequest() {
+    // In these tests we'll generate errors, which triggers a lot of output from
+    // the log. Let's silence that.
+    jest.spyOn(process.stdout, 'write').mockImplementation(() => {});
+
+    const agent = supertest(createApp().callback());
+    return agent.post('/compressed-store').type('text');
+  }
+
+  it('returns an error when no `accept` header is specified', async () => {
+    const req = getPreconfiguredRequest();
+    await req.send('a').expect(400);
```

Comment:
I think it would be beneficial to snapshot these message bodies, so we can tell if they change in the future.